log4j-core/src/main/java/org/apache/logging/log4j/core/pattern/AnsiEscape.java

```diff
@@ -320,12 +320,42 @@ public static String createSequence(final String... names) {
         boolean first = true;
         for (final String name : names) {
             try {
-                final AnsiEscape escape = EnglishEnums.valueOf(AnsiEscape.class, name.trim());
                 if (!first) {
                     sb.append(AnsiEscape.SEPARATOR.getCode());
                 }
                 first = false;
-                sb.append(escape.getCode());
+                String hexColor = null;
+                final String trimmedName = name.trim();
```

Comment:
```suggestion
                final String trimmedName = name.trim().toUpperCase(Locale.ENGLISH);
```

since the other keywords are also matched in a case insensitive way.